### PR TITLE
reword warning about `equals` and cooperative equality

### PR DESCRIPTION
**Pull request by SethTisue**
this is a small adjustment to @eed3si9n's #8120

it's no big deal, but the old wording kind of made cooperative equality sound like what we need rescuing from, rather than what will rescue us

### CODE REVIEW

**Review by som-snytt — APPROVED** [review]
`uses uncooperative equality`

Uncooperative equality describes the state of democracy in the US.

File: src/compiler/scala/tools/nsc/typechecker/RefChecks.scala [hunk 1]
```diff
@@ -1148,7 +1148,7 @@ abstract class RefChecks extends Transform {
       val actual   = underlyingClass(other.tpe)
       def typesString = "" + normalizeAll(qual.tpe.widen) + " and " + normalizeAll(other.tpe.widen)
       def nonSensiblyEquals() = {
-        refchecksWarning(pos, s"comparing values of types $typesString using `${name.decode}` is unsafe due to cooperative equality; use `==` instead", WarningCategory.OtherNonCooperativeEquals)
+        refchecksWarning(pos, s"comparing values of types $typesString using `${name.decode}` unsafely bypasses cooperative equality; use `==` instead", WarningCategory.OtherNonCooperativeEquals)
```
Comment (eed3si9n) [inline src/compiler/scala/tools/nsc/typechecker/RefChecks.scala:1151]: Fine by me, but I guess my ideal is `==` become the symbolic alias of `equals` and we take down the patriarchy of so-called _cooperative equality_. A system that is closed to the participation of user-land extension, like `UInt` type by Spire, directly goes against the philosophy of growable language, which Scala is named after.

**Comment by eed3si9n** [comment]
> it's no big deal, but the old wording kind of made cooperative equality sound like what we need rescuing from, rather than what will rescue us

You read it right. The crony capitalism is corrupt. We need equal protection under eq law - https://eed3si9n.com/equal-protection-under-eq-law

**Comment by SethTisue** [comment]
ah, now your efforts to radicalize Scala users through propagandistic warning messages are exposed! nice try, troublemaker!! 💣

File: src/compiler/scala/tools/nsc/typechecker/RefChecks.scala (hunk 1 above)
Comment (som-snytt) [reply src/compiler/scala/tools/nsc/typechecker/RefChecks.scala:1151]: Let's seize the moment to rename the language to `Growa`. Then you can say, Grow a `UInt`. And the t-shirt can read, Grow a `Pair`.

**Pull request merged**